master/reference/felix/configuration.md

```diff
@@ -118,4 +119,4 @@ to set a per-host override for "myhost" to move the log file to /tmp/felix.log:
 
 (For a global setting, omit the `--node=` option.)
 
-For more information, see the [calicoctl config documentation](../calicoctl/commands/config).
```

Comment:
Yeah it was unintentional. I can remove it if desired. My editor adds a line break at the end of the file if there is not one.